daprdocs/content/en/concepts/components-concept.md

```diff
@@ -14,18 +14,34 @@ Dapr uses a modular design where functionality is delivered as a component. Each
 
  The following are the component types provided by Dapr:
 
-* [Bindings](https://github.com/dapr/components-contrib/tree/master/bindings)
-* [Pub/sub](https://github.com/dapr/components-contrib/tree/master/pubsub)
-* [Middleware](https://github.com/dapr/components-contrib/tree/master/middleware)
+* [State stores](https://github.com/dapr/components-contrib/tree/master/state)
```

Comment:
While we're at it, should these just all link to the relevant component reference section in the docs?

Reply:
Good idea. Updated